packages/ckeditor5-special-characters/src/ui/specialcharactersnavigationview.js

```diff
@@ -43,7 +43,6 @@ export default class SpecialCharactersNavigationView extends FormHeaderView {
 		 * @member {module:ui/dropdown/dropdownview~DropdownView}
 		 */
 		this.groupDropdownView = this._createGroupDropdown( groupNames );
-		this.groupDropdownView.panelPosition = locale.uiLanguageDirection === 'rtl' ? 'se' : 'sw';
```

Comment:
Probably this is unnecessary since the dropdown handles its position correctly on its own.
Besides, it doesn't seem to work. Shouldn't it be set by the `set` method?